Brass-Demons/Welcome-to-Nashville
# Ms itinerary

# Description

Added meetup event ids from dummy api to save button class names, added event listener to the results container, targeted save button and its associated information and appened it to the itinerary object and the DOM


## Type of change

Please delete options that are not relevant.




# Testing Instructions

Make sure event listeners are functioning correctly and meetup information is populating to the DOM in the itinerary section

- Pull down the feature branch with `git fetch --all`.
- Checkout into the branch to review the feature on your local server.

# Checklist:
